platforms/documentation/docs/src/docs/userguide/dep-man/publishing/publishing_maven.adoc

```diff
@@ -159,6 +158,31 @@ You can define multiple repositories as long as they have unique names within th
 
 You can also configure any authentication details that are required to connect to the repository. See link:{groovyDslPath}/org.gradle.api.artifacts.repositories.MavenArtifactRepository.html[MavenArtifactRepository] for more details.
 
+[[supported_maven_repositories]]
+=== Supported Repositories
+
+Gradle's `maven-publish` plugin is compatible with the link:https://maven.apache.org/plugins/maven-deploy-plugin/[Maven Deploy Plugin] and supports publishing to any repository that adheres to the same standards and protocols.
+This includes a wide range of Maven-compatible repositories, such as:
+
+* **Local repositories** (e.g. `~/.m2/repository`)
+* **Custom remote Maven repositories**
+* **Repository managers** like **Sonatype Nexus** and **JFrog Artifactory**
+* **GitHub Packages**
+* **Internal corporate Maven repositories**
+
+[[maven_central]]
+=== Publishing to Maven Central
+
+The service commonly known as *Maven Central*, operated by Sonatype and officially hosted at link:https://central.sonatype.com/[central.sonatype.com] is a widely used repository for distributing Java and Kotlin libraries.
```

Comment:
I don't understand what "officially hosted" means. Can it be hosted unofficially?

Reply:
Yes, works for me

> 💅 We may want to add the more widely known URLs from https://central.sonatype.org/consume/

We have a Cookbook page for that. So no need to add them here, can be done later